curl_cffi/__init__.py

```diff
@@ -14,11 +14,9 @@
 ]
 
 import _cffi_backend  # noqa: F401  # required by _wrapper
-# This line includes _wrapper.so into the wheel
-from ._wrapper import ffi, lib  # type: ignore
 
-from .const import CurlInfo, CurlMOpt, CurlOpt, CurlECode, CurlHttpVersion, CurlWsFlag
-from .curl import Curl, CurlError, CurlMime
+# This line includes _wrapper.so into the wheel
+from ._wrapper import ffi, lib
 from .aio import AsyncCurl
-
-from .__version__ import __title__, __version__, __description__, __curl_version__
```

Comment:
This line is needed for downstream packages and users.

Reply:
Done